index.html

```diff
@@ -43,10 +51,18 @@ <h2 class="occupation">I’am a Software Developer</h2>
     </section>
 
     <section class="mid-part">
-        <h2 class="recentwork">My Recent Works</h2>
+            <div class="recent-title">
+                <h2 class="recentwork">My Recent Works</h2>
+                <hr class="hrrecent">
+            </div>
+            <!-- <div class="hrrecent">
+                <hr>
+            </div> -->
+        
         <div class="grid-container">
             <div class="multi">
```

Comment:
Your design             |  Figma Design
:-------------------------:|:-------------------------:
![badwidth](https://user-images.githubusercontent.com/69638013/154551883-723cee24-b116-4161-926c-ae2e9bcd19a2.PNG)  | ![width](https://user-images.githubusercontent.com/69638013/154551856-ac5d97db-9a47-423b-803f-cebb0b03b854.PNG)


- [ ] Kindly align this multi div to look like the original design, check the screenshot above 
- [ ] Kindly adjust second div to start where the image starts 🤗